bids2table/_b2t.py

```diff
@@ -56,11 +57,14 @@ def bids2table(
     root = Path(root).expanduser().resolve()
     if not root.is_dir():
         raise FileNotFoundError(f"root directory {root} does not exists")
-
+    
+    if exclude is None:
+        exclude = []
+        
     source = Crawler(
         root=root,
         include=["sub-*"],  # find subject dirs
-        skip=["sub-*"],  # but don't crawl into subject dirs
+        skip=["sub-*"] + exclude,  # but don't crawl into subject dirs
```

Comment:
Probably we should also pass `exclude` to the `Crawler`'s own `exclude` arg? But this should only matter in the odd case that someone tries to exclude a particular subject directory (which to be fair I'm not sure why anyone would do).